workspaces/copilot/packages/app/src/App.tsx

```diff
@@ -96,10 +100,11 @@ const routes = (
     </Route>
     <Route path="/settings" element={<UserSettingsPage />} />
     <Route path="/catalog-graph" element={<CatalogGraphPage />} />
-    <Route path="/copilot" element={<CopilotPage />} />
+    <Route path="/copilot" element={<CopilotHomePage />} />
```

Comment:
You could better achieve this via a routable component and sub routes, where the routes are handled within the plugin itself rather than needing to be added in the consumer frontend.

You can take a look at the docs here https://backstage.io/docs/plugins/composability/#extensions

Reply:
Thanks for the suggestion, @ScottGuymer! I tried this a few times before, but it didn't work because I had a different version of react-router installed. Now, with this approach of routable components and sub-routes, everything works perfectly! Thank you so much for your help!